src/js/jwt/insights/user.js

```diff
@@ -1,4 +1,11 @@
 const log = require('../logger')('insights/user.js');
+const servicesApi = require('./entitlements');
+const pathMapper = {
+    rhel: 'smart_management',
+    hybrid: 'hybrid_cloud',
+    insights: 'insights',
+    uhc: 'openshift'
```

Comment:
do we want to change this to `openshift` instead of `uhc`?

Reply:
Oh yeah, good point now that routes has changed.